FiveBoard.html

```diff
@@ -1,87 +1,92 @@
+        </p>
+        <img
+          id="hint-img"
+          class="game-icon"
+          src="images/08_X-Icon.png"
+          alt="hint image"
+        />
+      </div>
+      <div class="selection-options">
+        <a id="hint-close" class="nav-link" href="javascript:void(0)">
+          <div class="load-item initial-load">x</div>
+        </a>
+      </div>
+    </div>
 
-	<!-- Start new game pop up box -->
-	<div id="newgame-dialog" class="pop-up-box load-dialog">
-		<div class="text-box load-section">
-			<p class="text-content">Start a new game?</p>
-		</div>
-		<div class="selection-options">
-			<a id='new-game-yes' class="nav-link" href="javascript:void(0)">
-				<div class="menu-title load-item initial-load">Yes
-				</div>
-			</a>
-			<a id='new-game-no' class="nav-link" href="javascript:void(0)">
-				<div class="menu-title load-item initial-load">No
-				</div>
-			</a>
-		</div>
-	</div>
+    <!-- Start new game pop up box -->
+    <div id="newgame-dialog" class="pop-up-box load-dialog">
+      <div class="text-box load-section">
+        <p class="text-content">Start a new game?</p>
+      </div>
+      <div class="selection-options">
+        <a id="new-game-yes" class="nav-link" href="javascript:void(0)">
+          <div class="menu-title load-item initial-load">Yes</div>
+        </a>
+        <a id="new-game-no" class="nav-link" href="javascript:void(0)">
+          <div class="menu-title load-item initial-load">No</div>
+        </a>
+      </div>
+    </div>
 
-	<!-- CSS Preloader -->
-	<div id="preloader" class="load-wrapper">
-		<div class="loader-back"></div>
-		<div id="loader-box">
-			<div class="loader-ball">
-		      <div class="ball ball-first" id="ball-1"></div>
-		    </div>
-		    <div class="loader-ball">
-		      <div class="ball ball-second" id="ball-2"></div>
-		    </div>
-		    <div class="loader-ball">
-		      <div class="ball ball-first" id="ball-3"></div>
-		    </div>
-		    <div class="loader-ball">
-		      <div class="ball ball-second" id="ball-4"></div>
-		    </div>
-		</div>
-	</div>
+    <!-- CSS Preloader -->
+    <div id="preloader" class="load-wrapper">
+      <div class="loader-back"></div>
+      <div id="loader-box">
+        <div class="loader-ball">
+          <div class="ball ball-first" id="ball-1"></div>
+        </div>
+        <div class="loader-ball">
+          <div class="ball ball-second" id="ball-2"></div>
+        </div>
+        <div class="loader-ball">
+          <div class="ball ball-first" id="ball-3"></div>
+        </div>
+        <div class="loader-ball">
+          <div class="ball ball-second" id="ball-4"></div>
+        </div>
+      </div>
+    </div>
+  </body>
 
-</body>
-
-<script type="text/javascript" src="js/preload.js"></script>
-<script type="module" src="js/temp.js"></script>
-
-</html>
\ No newline at end of file
+  <script type="text/javascript" src="js/preload.js"></script>
+  <script type="module" src="js/temp.js"></script>
+</html>

```

Comment:
Nitpick: The changes here are only formatting-related.

Reply:
I'm guessing I should change my default indentation to 2 spaces rather than 4.